aws/v15.0.0/README.md

```diff
@@ -0,0 +1,1066 @@
+# :zap: Giant Swarm Release v15.0.0 for AWS :zap:
+
+This release provides support for Kubernetes 1.20 on AWS. It also enables the Container Storage Interface (CSI) and the automatic termination of unhealthy nodes by default.
```

Comment:
We should also mention storage class `gp3` is becoming the standard by upgrading, before it was `gp2`

Reply:
I updated the highlights section to mention that `gp3` is now the default storage class.